### ISSUE TITLE
Keep displaying highway tunnels when using hide underground objects

### ISSUE THREAD

**Issue by naoliv** [osmandapp/Osmand#3561]
With the default map style, when selecting `Configure map` → `Hide` and selecting `Underground objects`, highway tunnels are also hidden.

For example, here https://www.openstreetmap.org/#map=19/-21.79347/-48.17254

![](https://cloud.githubusercontent.com/assets/982338/24076235/7eaefad8-0c0a-11e7-99f0-b726ca917cde.png)

Couldn't we have an exception for highway tunnels and keep displaying them?
When I ask to hide underground objects, I would like to not see subways and other kind of objects; highway tunnels are still interesting to have on the map.

**Comment by vshcherb** [comment]
Hard to say though I agree cutting tunnels breaks the road network and creates some "artificial" map.

**Comment by xmd5a2** [comment]
What to do with footway tunnels? They are widely used in subway stations.

**Comment by vshcherb** [comment]
Good point, I will put that issue on hold cause it is not main functionality (not enabled by default).
